R/rowClust.R

```diff
@@ -0,0 +1,41 @@
+# REQUIRED PACKAGES: "fastcluster", "stats", "coop"
+rowClust <- function(tse,
+                     abund_values = "counts",
+                     metric = "spearman", # c("pearson", "kendall", "spearman" || 
+                                          #    "cosine" ||
+                                          #    "euclidean", "maximum", "manhattan", "canberra", "binary", "minkowski")
+                     method = "ward.D2", # c("single", "complete", "average", "mcquitty", "ward.D", "ward.D2", "centroid","median")
+                     name = "clust_id",
+                     k = NULL, # number of clusters
+                     h = NULL, # height of cut-off
+                     ... # arguments passed to stats::dist (mainly for the p arg of "minkowski")
+                     ) {
+  # Getting Assay data
```

Comment:
Good strat! Soem suggestions: Please check some other functions in `mia` on how checks are done for inputs.       
https://github.com/microbiome/mia/blob/master/R/estimateDivergence.R  
```
################### Input check ###############
        # Check abund_values
        .check_assay_present(abund_values, x)
        # Check name
        if(!.is_non_empty_character(name) || length(name) != 1L){
            stop("'name' must be a non-empty character value.",
                 call. = FALSE)
        }
```

`k` and `h` must be either numeric or NULL, if NULL or not a single numeric value it should return an error

Reply:
Thanks, I will take a look and fix that.